### PR TITLE
reuse annotation from api

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**Please provide a description of this PR:**

### CODE REVIEW

File: pilot/pkg/autoregistration/controller.go [hunk 1]
```diff
@@ -101,7 +91,7 @@ type Controller struct {
 	// do not support writing. We only use it here for reading WorkloadEntry/WorkloadGroup.
 	store model.ConfigStoreController
 
-	// Note: unregister is to update the workload entry status: like setting `DisconnectedAtAnnotation`
+	// Note: unregister is to update the workload entry status: like setting `annotation.IoIstioDisconnectedAt.Name`
```
Comment (ramaraochavali) [inline pilot/pkg/autoregistration/controller.go:94]: nit: comment with actual annotation name is more readable

File: pilot/pkg/autoregistration/controller.go [hunk 2]
```diff
@@ -72,17 +73,6 @@ var (
 )
 
 const (
-	// TODO use status or another proper API instead of annotations
-
-	// AutoRegistrationGroupAnnotation on a WorkloadEntry stores the associated WorkloadGroup.
-	AutoRegistrationGroupAnnotation = "istio.io/autoRegistrationGroup"
```
Comment (ramaraochavali) [inline pilot/pkg/autoregistration/controller.go:78]: May be it is lot simpler to replace the name at this constant and use the constants every where?

File: pilot/pkg/autoregistration/controller.go (hunk 1 above)
Comment (hzxuzhonghu) [reply pilot/pkg/autoregistration/controller.go:94]: Yes, but just align with others

File: pilot/pkg/autoregistration/controller.go (hunk 2 above)
Comment (hzxuzhonghu) [reply pilot/pkg/autoregistration/controller.go:78]: Yes, but just align with others

File: pilot/pkg/autoregistration/controller.go (hunk 1 above)
Comment (hzxuzhonghu) [reply pilot/pkg/autoregistration/controller.go:94]: OK, will restore

**Review by howardjohn — APPROVED** [review]

**Pull request merged**